build/import/Packages.targets

```diff
@@ -14,15 +14,18 @@
       https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json;
       https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json;
       https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json;
+      https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json;
       https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;
       https://api.nuget.org/v3/index.json;
-      https://dotnet.myget.org/F/symreader-converter/api/v3/index.json
+      https://dotnet.myget.org/F/symreader-converter/api/v3/index.json;
     </RestoreSources>
   </PropertyGroup>
 
   <ItemGroup>
 
     <!-- Toolset -->
+    <PackageReference Include="Nerdbank.Streams"                                                      Version="2.6.81" />
```

Comment:
Please use the same pattern as all the other package references. Update goes in this file, the reference goes in the layer appropriate file (VisualStudio.props, etc).

Reply:
done